scripts/run_mypy_checks.py

```diff
@@ -769,37 +774,65 @@ def install_third_party_libraries(skip_install):
         install_third_party_libs.main()
 
 
-def get_mypy_cmd(files):
+def get_mypy_cmd(files, ci_check):
     """Return the appropriate command to be run.
 
     Args:
         files: list(list(str)). List having first element as list of string.
+        ci_check: bool. Whether the test is for CI.
 
     Returns:
         list(str). List of command line arguments.
     """
+    # TODO(#13113): Change mypy command to mypy path after Python3 migration.
+    if ci_check:
+        mypy_cmd = 'mypy'
+    else:
+        mypy_cmd = os.path.join(
+            os.getcwd(), 'third_party', 'python3_libs', 'bin', 'mypy')
     if files:
-        cmd = [MYPY_CMD, '--config-file', CONFIG_FILE_PATH] + files
+        cmd = [mypy_cmd, '--config-file', CONFIG_FILE_PATH] + files
     else:
         excluded_files_regex = (
             '|'.join(NOT_FULLY_COVERED_FILES + EXCLUDED_DIRECTORIES))
         cmd = [
-            MYPY_CMD, '--exclude', excluded_files_regex,
+            mypy_cmd, '--exclude', excluded_files_regex,
             '--config-file', CONFIG_FILE_PATH, '.'
         ]
     return cmd
 
 
-def install_mypy_prerequisites():
+def install_mypy_prerequisites(ci_check):
```

Comment:
```suggestion
def install_mypy_prerequisites(install_globally):
```

The param name shoukd explain the usage*

Reply:
Done.

Reply:
I don't see this fixed in the current diff

Reply:
Fixed